saleor/graphql/order/mutations/draft_order_create.py

```diff
@@ -345,10 +346,22 @@ def save(cls, info, instance, cleaned_input):
     def _save_draft_order(
         cls, info, instance, cleaned_input, *, is_new_instance, app, manager
     ):
+        updated_fields = []
         with traced_atomic_transaction():
             # Process addresses
             cls._save_addresses(instance, cleaned_input)
 
+            # Process shipping method
+            method = cleaned_input["shipping_method"]
+            if method:
+                shipping_channel_listing = cls.validate_shipping_channel_listing(
+                    method, instance
+                )
+                cls.update_shipping_method(instance, method, shipping_channel_listing)
+                updated_fields.extend(SHIPPING_METHOD_UPDATE_FIELDS)
+            else:
+                cls.clean_shipping_method_from_order(instance)
```

Comment:
It should be called `clear_shipping_method_from_order` IMO, because "clean" refers to a different part of mutations flow, where we're "cleaning" input data for further processing.

Reply:
Now it will clear the shipping method when it's not provided in the input, right? Is that expected? I guess we should clear it only when the shipping method is explicitly set to `null` in the input.

Reply:
It is fixed now. Shipping method will be cleared when it's explicitly set as `None` in input. Also changed the name.